addons/mail/static/src/components/thread_view/thread_view.js

```diff
@@ -121,11 +123,18 @@ class ThreadView extends Component {
         const threadView = this.env.models['mail.thread_view'].get(props.threadViewLocalId);
         const thread = threadView ? threadView.thread : undefined;
         const threadCache = threadView ? threadView.threadCache : undefined;
+        const correspondent = thread && thread.correspondent;
         return {
+            composer: thread && thread.composer,
+            correspondentId: correspondent && correspondent.id,
             isDeviceMobile: this.env.messaging.device.isMobile,
-            thread: thread ? thread.__state : undefined,
-            threadCache: threadCache ? threadCache.__state : undefined,
-            threadView: threadView ? threadView.__state : undefined,
+            thread,
+            threadCacheIsLoaded: threadCache && threadCache.isLoaded,
+            threadIsTemporary: thread && thread.isTemporary,
+            threadMassMailing: thread && thread.mass_mailing,
+            threadModel: thread && thread.model,
+            threadView,
+            threadViewIdLoading: threadView && threadView.isLoading,
```

Comment:
typo on key (doesn't really matter though...)

Reply:
Indeed, I'll still fix it. Having an object is useless though, I guess it would be less verbose if we converted those to array, as we had during a short period way back then?